container/example-k8s.dockerfile

```diff
@@ -9,17 +16,29 @@ WORKDIR /opt/hivemq
 RUN groupadd --gid ${HIVEMQ_GID} hivemq \
     && useradd -g hivemq -d /opt/hivemq -s /bin/bash --uid ${HIVEMQ_UID} hivemq
 
-COPY --from=hivemq/hivemq4-test:k8s-snapshot --chown=hivemq:hivemq /opt/hivemq /opt/hivemq
-COPY --from=hivemq/hivemq4-test:k8s-snapshot --chown=hivemq:hivemq /opt/*.sh /opt/
-COPY --from=hivemq/hivemq4-test:k8s-snapshot --chown=hivemq:hivemq /docker-entrypoint.d /docker-entrypoint.d/
+COPY --from=hivemq_image --chown=hivemq:hivemq /opt/hivemq /opt/hivemq
+COPY --from=hivemq_image --chown=hivemq:hivemq /opt/*.sh /opt/
+COPY --from=hivemq_image --chown=hivemq:hivemq /docker-entrypoint.d /docker-entrypoint.d/
 
 RUN apt-get update && apt-get install -y --no-install-recommends curl gnupg-agent gnupg unzip libnss-wrapper \
-  && apt-get purge -y gpg && apt-get clean && rm -rf /var/lib/apt/lists/* 
-RUN chmod g+w /opt/hivemq /opt/hivemq/extensions /opt/hivemq/conf /opt/hivemq/extensions/hivemq-prometheus-extension /opt/hivemq/extensions/hivemq-bridge-extension  /opt/hivemq/extensions/hivemq-dns-cluster-discovery  /opt/hivemq/extensions/hivemq-enterprise-security-extension  /opt/hivemq/extensions/hivemq-k8s-sync-extension  /opt/hivemq/extensions/hivemq-kafka-extension /opt/hivemq/extensions/hivemq-kafka-extension/DISABLED /opt/hivemq/extensions/hivemq-bridge-extension/DISABLED /opt/hivemq/extensions/hivemq-enterprise-security-extension/DISABLED  \
-    && chown hivemq:hivemq /opt/docker-entrypoint.sh /opt/hivemq \
+  && apt-get purge -y gpg && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Setup permisions to the same group to the configuration and extensions, we are explicit to don't oversize the image
+RUN chmod g+w /opt/hivemq /opt/hivemq/extensions /opt/hivemq/conf /opt/hivemq/extensions/hivemq-prometheus-extension \
```

Comment:
is it possible to use -R (recursive) on chmod here to shorten this line?

Reply:
yes, but the size of the image explodes